src/app.module.ts

```diff
@@ -1,14 +1,14 @@
-import { Module } from '@nestjs/common';
-import { ConfigModule } from '@nestjs/config';
-import { EventEmitterModule } from '@nestjs/event-emitter';
+import { Module } from "@nestjs/common";
```

Comment:
I think your VSCode is set to format with prettier which uses double quotes
Could you run `npm run lint` or set your workspace to use Eslint instead? This will ensure our code patterns for this repository.

Here is the config I use on VSCode, that eliminates the need to run `npm run lint` every time.

```json
{
  "[javascript]": {
    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
  },
  "[typescript]": {
    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
  },
  "editor.formatOnSave": true,
}
  ```